selection/utils.py

```diff
@@ -55,3 +59,120 @@ def get_utilized_indexes(
             }
 
     return utilized_indexes_workload, query_details
+
+
+# --- Join Order Benchmark utilities ---
+
+IMDB_LOCATION = "https://archive.org/download/imdb_20200624/imdb.zip"
+IMDB_FILE_NAME = "imdb.zip"
+IMDB_TABLE_DIR = "imdb_data"
+IMDB_TABLE_NAMES = [
+    "aka_name",
+    "aka_title",
+    "cast_info",
+    "char_name",
+    "company_name",
+    "company_type",
+    "comp_cast_type",
+    "complete_cast",
+    "info_type",
+    "keyword",
+    "kind_type",
+    "link_type",
+    "movie_companies",
+    "movie_info",
+    "movie_info_idx",
+    "movie_keyword",
+    "movie_link",
+    "name",
+    "person_info",
+    "role_type",
+    "title",
+]
+
+
+def _clean_up(including_table_dir=False):
+    if os.path.exists(IMDB_FILE_NAME):
+        os.remove(IMDB_FILE_NAME)
+
+    if including_table_dir and os.path.exists(IMDB_TABLE_DIR):
+        for file in os.listdir(IMDB_TABLE_DIR):
+            os.remove("./%s/%s" % (IMDB_TABLE_DIR, file))
+        os.rmdir(IMDB_TABLE_DIR)
+
+
+def _files_exist():
+    for table_name in IMDB_TABLE_NAMES:
+        if not os.path.exists(os.path.join(IMDB_TABLE_DIR, table_name + ".csv")):
+            return False
+
+    return True
+
+
+def download_and_uncompress_imdb_data():
```

Comment:
Looks ok to me (without looking into details or whether there is a shorter way)

Reply:
This is actually copied from https://github.com/hyrise/hyrise/blob/master/scripts/setup_imdb.py